ucm2/Qualcomm/sc8280xp/SC8280XP-LENOVO-X13S.conf

```diff
@@ -0,0 +1,10 @@
+Syntax 4
```

Comment:
Filename - double sc8280xp ? LENOVO-X13S should be enough.

Also, I would create sc8280xp.conf and do the redirect to LENOVO-X13S.conf there. See USB-Audio.conf and `${sys:devices/virtual/dmi/...` usage (like in AMD/acp5x/acp5x.conf). I would expect that we may need to describe other hardware types in future.

Reply:
Will remove  "SC8280XP-"  prefix, I already have a sc8280xp.conf link in .conf.d to point to Qualcomm/sc8280xp/LENOVO-X13S.conf 
you mean something like this in LENOVO-X13S.conf

If.ThinkpadX13s {
        Condition {
                Type String
                String1 "X13s"
                String2 "${sys:devices/virtual/dmi/id/product_family}"
        }
        True {
                SectionUseCase."HiFi" {
                        File "/Qualcomm/sc8280xp/HiFi.conf"
                        Comment "HiFi quality Music."
                }
                Include.card-init.File "/lib/card-init.conf"
                Include.ctl-remap.File "/lib/ctl-remap.conf"
                Include.codec-init.File "/codecs/wcd938x/init.conf"
   }
}

Reply:
I would add also vendor check:

```
                String1 "LENOVO-X13s"
                String2 "${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family}"
```

Reply:
sure, will try that out and update the PR.

Reply:
Once we have below checks in  ucm2/Qualcomm/sc8280xp/sc8280xp.conf  then I feel checks in LENOVO-X13S.conf are totally redundant. Or do you still feel the need of more checks in ucm2/Qualcomm/sc8280xp/LENOVO-X13S.conf ?

ucm2/Qualcomm/sc8280xp/sc8280xp.conf:
Syntax 4

If.LENOVOX13s {
	Condition {
                Type RegexMatch
                String "${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family}"
                Regex "LENOVO.*ThinkPad X13s.*"
        }
	True.Include.x13s.File "/Qualcomm/sc8280xp/LENOVO-X13s.conf"
	False.Error "SC8280XP - ${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family} model not supported"
}

Reply:
thanks for confirming I think I have addressed all the changes as discussed and updated the PR.